CHANGELOG.md

```diff
@@ -2,160 +2,178 @@
   their own, dedicated `keyserver` command. Run `singularity help keyserver` for
   more information.
+
+#### CLI
+
 - The commands related to OCI/Docker registries that were under `remote` have
   been moved to their own, dedicated `registry` command. Run
   `singularity help registry` for more information.
-- The the `remote list` subcommand now outputs only remote endpoints (with
+- The `remote list` subcommand now outputs only remote endpoints (with
   keyservers and OCI/Docker registries having been moved to separate commands),
   and the output has been streamlined.
+- `--cwd` is now the preferred form of the flag for setting the container's
+  working directory, though `--pwd` is still supported for compatibility.
 - Adding a new remote endpoint using the `singularity remote add` command will
   now set the new endpoint as default. This behavior can be suppressed by
   supplying the `--no-default` (or `-n`) flag to `remote add`.
-- Singularity uses squashfuse_ll/squashfuse, which is now built from a git
-  submodule unless `--without-squashfuse` is specified as an argument to
-  `mconfig`. When built with `--without-squashfuse`, `squashfuse_ll` or
-  `squashfuse` will be located on `PATH`. Version 0.2.0 or later is required.
-- When `singularity registry login` is used to login to a public OCI registry,
-  the password is not verified at login time.
 - Improved the clarity of `singularity key list` output.
-- Do not mount current working directory when its path in the container and on
-  the host contain symlinks to different locations.
-- Create current working directory in container when it doesn't exist, so that
-  it can be entered. You must now specify `--no-mount home,cwd` instead of just
-  `--no-mount home` to avoid mounting from `$HOME` if you run `singularity` from
-  inside `$HOME`.
-- Running `singularity pull --oci` without a specific output filename will now
-  create an image with the `.oci.sif` suffix instead of `.sif`.
+
+#### Runtime Behaviour
+
+- The way `--home` is handled when running as root (e.g. `sudo singularity`) or
+  with `--fakeroot` has changed. Previously, we were only modifying the `HOME`
+  environment variable in these cases, while leaving the container's
+  `/etc/passwd` file unchanged (with its homedir field pointing to `/root`,
+  regardless of the value passed to `--home`). With this change, both value of
+  `HOME` and the contents of `/etc/passwd` in the container will reflect the
+  value passed to `--home`.
+- Bind mounts are now performed in the order of their occurrence on the command
+  line, or within the value of the `SINGULARITY_BIND` environment variable.
+  (Previously, image-mounts were always performed first, regardless of order.)
+- Default OCI config generated with `singularity mount` no longer sets any
+  inheritable / ambient capabilites, matching other OCI runtimes.
+- `singularity oci mount` now uses, and requires, `squashfuse` to mount a SIF
```

Comment:
Should we change this sentence slightly in light of the `squashfuse_ll`/`squashfuse` flexibility?